### DISCUSSION TITLE
zh/guide/markdown/code-tabs

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#434] [Announcements]
# zh/guide/markdown/code-tabs

主题为你带来了代码块分组支持。

https://theme-hope.vuejs.press/zh/guide/markdown/code-tabs.html

**Comment by kdyzm** [comment]
请问，代码中某几行特殊标记出来是怎么做到的，就像本文章模块中第一个代码块中的八到十行代码

**Reply by qiaoshengchuan** [reply]
https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/main/docs/theme/src/zh/guide/markdown/code-tabs.md?plain=1
看本页的写法是：```ts {8-10}
// .vuepress/config.ts
import { defineUserConfig } from "vuepress";
import { hopeTheme } from "vuepress-theme-hope";
